ServerCore/Pages/Teams/Edit.cshtml

```diff
@@ -65,12 +65,15 @@
             </div>
             @if(Model.Event.AllowsRemoteTeams)
             {
-                <div class="form-group">
-                    <label asp-for="Team.IsRemoteTeam" class="control-label">Team is playing remotely</label>
+                if(!Model.Event.LockChangesToRemoteStatus)
```

Comment:
Since there are no changes to the Edit.cshtml.cs file, I assume that I could get around remote/local limits by declaring my team in the bucket that has room and then editing the team to toggle this flag.

Reply:
This hides the UI for making that edit and then if the player isn't an admin they'll bounce off of the change even if they do it through F12 or something since only admins can ignore the team caps.

Reply:
If you're resolving this because for PH24 the flag is true thus nothing can happen in the next week, fine, but unless I'm fundamentally misunderstanding things, this needs to be fixed eventually.